src/Wizard/Wizard.jsx

```diff
@@ -18,22 +18,31 @@ export class Wizard extends React.Component {
     this.state = _.assign(INITIAL_STATE, {
       data: props.initialWizardData || {},
     });
+
     this.reset = this.reset.bind(this);
     this.prevStepHandler = this.prevStepHandler.bind(this);
     this.nextStepHandler = this.nextStepHandler.bind(this);
     this.calculatePercentComplete = this.calculatePercentComplete.bind(this);
+
+    if (this.props.initialStep) {
+      this.state = Object.assign({}, this.state, this._getStateAtStep(this.props.initialStep));
```

Comment:
This line can just be written `Object.assign(this.state, this._getStateAtStep(this.props.initialStep))` – `Object.assign` mutates the first parameter.